golang/cosmos/Makefile

```diff
@@ -13,8 +13,12 @@ whitespace += $(whitespace)
 comma := ,
 build_tags_comma_sep := $(subst $(whitespace),$(comma),$(build_tags))
 
+# Note that the version.Name must be alphanumeric only.
+# Otherwise, generated "os" keyrings on Ubuntu 20.04 can't be read.
+VersionName := $(shell echo "$(NAME)" | sed -e 's/[^A-Za-z0-9]//g')
```

Comment:
Good place for `tr -d` too.

Reply:
I guess this is stylistic.

For me, being able to forget `tr`'s speciali*sed* flags (hint: `tr -d` does not do the job) mitigates the cognitive load of using `sed` for everything. :smile:

Reply:
Fair. I use `perl` where `awk` shines best for the same reason.